src/components/VaultRow/lib/VaultDepositForm.tsx

```diff
@@ -32,6 +35,15 @@ export const VaultDepositForm: FC<VaultProps> = (props) => {
   const { address: userAddress } = useAccount()
   const chainId = useActiveChainId()
   const vault = useVault(props)
+  const toastManager = useToast()
+  const [depositToastId, setDepositToastId] = useState<string | undefined>()
+  const [approveToastId, setApproveToastId] = useState<string | undefined>()
+  const depositLoadingMsg = "Waiting for deposit transaction..."
```

Comment:
No need to have variables for these -- just define the message inline when invoking the toasts